doc/releases/changelog-0.41.0.md

```diff
@@ -220,323 +240,494 @@ With `qml.decompositions.enable_graph()`, the following new features are availab
+
+* It's now possible to use [Qualtran](https://qualtran.readthedocs.io/en/latest/) bloqs in PennyLane
+  with the new :func:`qml.FromBloq <pennylane.FromBloq>` class. 
+  [(#7148)](https://github.com/PennyLaneAI/pennylane/pull/7148)
+  
+  :func:`qml.FromBloq <pennylane.FromBloq>` translates [Qualtran bloqs](https://qualtran.readthedocs.io/en/latest/bloqs/index.html#bloqs-library) 
+  into equivalent PennyLane operators. It requires two inputs:
+  * `bloq`: an initialized Qualtran Bloq
+  * `wires`: the wires the operator acts on
+  
+  The following example applies a PennyLane Operator and Qualtran Bloq in the same circuit:
 
-* Added class ``qml.FromBloq`` that takes Qualtran bloqs and translates them into equivalent PennyLane operators. For example, we can now import Bloqs and use them in a way similar to how we use PennyLane templates:
   ```python
   >>> from qualtran.bloqs.basic_gates import CNOT
   
-  >>> dev = qml.device("default.qubit") # Execute on device
+  >>> dev = qml.device("default.qubit")
   >>> @qml.qnode(dev)
   ... def circuit():
+  ...    qml.X(wires=0)
   ...    qml.FromBloq(CNOT(), wires=[0, 1])
   ...    return qml.state()
   >>> circuit()
-  array([1.+0.j, 0.+0.j, 0.+0.j, 0.+0.j])
+  array([0.+0.j, 0.+0.j, 0.+0.j, 1.+0.j])
   ```
+
+* A new function called :func:`qml.bloq_registers <pennylane.bloq_registers>` is available to help
+  determine the required wires for complex Qualtran ``Bloqs`` with multiple registers.
   [(#7148)](https://github.com/PennyLaneAI/pennylane/pull/7148)
+  
+  Given a Qualtran Bloq, this function returns a dictionary of register names and wires.
 
-* ``qml.structure_constants`` now accepts and outputs matrix inputs using the ``matrix`` keyword.
-  [(#6861)](https://github.com/PennyLaneAI/pennylane/pull/6861)
+  ```pycon
+  >>> from qualtran.bloqs.phase_estimation import RectangularWindowState, TextbookQPE
+  >>> from qualtran.bloqs.basic_gates import ZPowGate
+  >>> textbook_qpe = TextbookQPE(ZPowGate(exponent=2 * 0.234), RectangularWindowState(3))
+  >>> registers = qml.bloq_registers(textbook_qpe)
+  >>> registers
+  {'q': Wires([0]), 'qpe_reg': Wires([1, 2, 3])}
+  ```
 
-<h4>Qualtran Integration 🔗</h4>
+  In the following example, we use these registers to measure the correct qubits in quantum phase
+  estimation:
+
+  ```python
+  dev = qml.device("default.qubit")
+  @qml.qnode(dev)
+  def circuit():
+      qml.FromBloq(textbook_qpe, wires=range(textbook_qpe.signature.n_qubits()))
+      return qml.probs(wires=registers['qpe_reg'])
+  ```
+
+  ```pycon
+  >>> print(qml.draw(circuit)())
+  0: ─╭FromBloq─┤       
+  1: ─├FromBloq─┤ ╭Probs
+  2: ─├FromBloq─┤ ├Probs
+  3: ─╰FromBloq─┤ ╰Probs
+  ```
+
+<h4>Hadamard Gradient Variants and Improvements 🌈</h4>
+
+* Variants of the Hadamard gradient outlined in [arXiv:2408.05406](https://arxiv.org/pdf/2408.05406) 
+  have been added as new differentiation methods: `diff_method="reversed-hadamard"`, `"direct-hamadard"`, 
```

Comment:
Happy to elaborate a bit more on these new diff methods if it's recommended :). Might be a good idea to convey to the user why they should consider using these, at least 😅 ...

As I'm typing this out, I think this should be elaborated on lol